docs/hub/spaces.md

```diff
@@ -87,6 +87,10 @@ pinned: false
 
 For additional settings, refer to the [Reference](#reference) section.
 
+### Compare or Clone Models and Spaces
+You can build a space comparing Hugging Face Models and Spaces or create clones of them with [Model Comparator Space Builder](https://huggingface.co/spaces/farukozderim/Model-Comparator-Space-Builder) 🤗.
```

Comment:
Maybe you can add a bit more info about this Space and split the information

Something like this maybe?
> 
> ### Comparing different models
> 
> There is a Space that allows you to automatically create a Space to compare different models and Spaces. ...
> 
> ### Duplicating a space
> 
> ...

Reply:
And I will create 2 different headings like you mentioned.

Reply:
Sounds good!

Reply:
updated the space and description, there is also a new heading that came into my mind :D